index.bs

```diff
@@ -495,6 +506,30 @@ through a process called [=sensor fusion=].
 [=calibration|Calibrated=] [=raw sensor readings=] are referred to as <dfn>sensor readings</dfn>,
 whether or not they have undergone [=sensor fusion=].
 
+If the [=sensor readings=] represent spatial measurements (e.g.
+acceleration, angular velocity), they must be resolved in
+a <dfn export>local coordinate system</dfn> that represents
+a reference frame for these [=sensor readings|readings=].
+A [=device sensor|sensor=] that provides such [=sensor readings|readings=]
+is referred to as <dfn export>spatial sensor</dfn>.
+
+Depending on the number of orthogonal axes in which the measurements
+can be simultaneously performed by a [=spatial sensor=], this
+[=spatial sensor|sensor=] can be <dfn>uniaxial</dfn>, <dfn>biaxial</dfn>,
+or <dfn>triaxial</dfn>.
+
+Scalar physical quantities (i.e. temperature) do not require
```

Comment:
Why call them scalar... a x value is also a scalar

Reply:
In a way, they are just values relative to a fixed scale. It gets confusing when we start talking about relative humidity, which is humidity relative to the current temperature. Maybe we could make this more clear

Reply:
Actually what you want it "Non-relativistic scalars"

Reply:
Non-relativistic scalar physical quantities such as temperature, time and mass, do not require a [=local coordinate system=] for resolution. Other scalars may have standardized frames of reference, such as relative humidity, which is relative to the current temperature.

Reply:
x is a vector component in a Cartesian coordinate system, i.e. the vector physical quantity is measured. 
Not sure why fixed scale is required, both temperature and relative humidity won't need a spatial coordinate system, isn't it?